openshift_scalability/nodeVertical.sh

```diff
@@ -16,17 +17,41 @@ long_sleep() {
 
 clean() { echo "Cleaning environment"; oc delete project clusterproject0; }
 
-# sleeping to gather some steady-state metrics, pre-test
-long_sleep
+golang_clusterloader() {
+  # Export kube config
+  export KUBECONFIG=${KUBECONFIG-$HOME/.kube/config}
+  MY_CONFIG=config/golang/nodeVertical
+  # loading cluster based on yaml config file
+  /usr/libexec/atomic-openshift/extended.test --ginkgo.focus="Load cluster" --viper-config=$MY_CONFIG
+}
 
-# loading cluster based on yaml config file
-./cluster-loader.py --file=$MY_CONFIG
+python_clusterloader() {
+  MY_CONFIG=config/nodeVertical.yaml
+  ./cluster-loader.py --file=$MY_CONFIG
+}
 
-# sleeping again to gather steady-state metrics after environment is loaded
+long_sleep() {
```

Comment:
Sorry, missed this before.  There are also duplicate long_sleep() in here.

Reply:
Sorry, didn't notice that I had the sleep function twice. Fixed.